smart-contracts/contracts/cl-vault/src/reply.rs

```diff
@@ -51,29 +42,41 @@ pub enum Replies {
     Unknown,
 }
 
-pub fn handle_reply(deps: DepsMut, env: Env, msg: Reply) -> Result<Response, ContractError> {
-    match msg.id.into() {
-        Replies::DepositCreatePosition => {
-            handle_deposit_create_position_reply(deps, env, msg.result)
-        }
-        Replies::InstantiateCreatePosition => {
-            handle_instantiate_create_position_reply(deps, env, msg.result)
-        }
-        Replies::CollectIncentives => handle_collect_incentives_reply(deps, env, msg.result),
-        Replies::CollectSpreadRewards => handle_collect_spread_rewards_reply(deps, env, msg.result),
-        Replies::WithdrawPosition => handle_withdraw_position_reply(deps, env, msg.result),
-        Replies::RangeInitialCreatePosition => {
-            handle_initial_create_position_reply(deps, env, msg.result)
-        }
-        Replies::RangeIterationCreatePosition => {
-            handle_iteration_create_position_reply(deps, env, msg.result)
-        }
-        Replies::Swap => handle_swap_reply(deps, env, msg.result),
-        Replies::Merge => handle_merge_response(deps, msg.result),
-        Replies::CreateDenom => handle_create_denom_reply(deps, msg.result),
-        Replies::WithdrawUser => handle_withdraw_user_reply(deps, msg.result),
-        Replies::WithdrawMerge => handle_merge_withdraw_reply(deps, env, msg.result),
-        Replies::CreatePositionMerge => handle_merge_create_position_reply(deps, env, msg.result),
-        Replies::Unknown => unimplemented!(),
-    }
+pub fn handle_create_denom_reply(
```

Comment:
Imo we should move these to instantiate.rs, since these functions are only used there and it keeps the same flow

Reply:
yes you are right. and transform them into not pub functions